programs/drift/src/math/liquidation.rs

```diff
@@ -486,3 +493,22 @@ pub fn get_liquidation_order_params(
 
     Ok(order_params)
 }
+
+pub fn get_liquidation_fee(
+    base_liquidation_fee: u32,
+    max_liquidation_fee: u32,
+    last_active_user_slot: u64,
+    current_slot: u64,
+) -> DriftResult<u32> {
+    let slots_elapsed = current_slot.safe_sub(last_active_user_slot)?;
+    if slots_elapsed < LIQUIDATION_FEE_ADJUST_GRACE_PERIOD_SLOTS {
+        return Ok(base_liquidation_fee);
+    }
+
+    let liquidation_fee = base_liquidation_fee.safe_add(
```

Comment:
just do saturating_add here just in case

Reply:
done